src/frontend/src/views/Data/components/SamplesView/components/SampleTableModalManager/components/DownloadModal/index.tsx

```diff
@@ -39,6 +41,9 @@ const DownloadModal = ({
   const [isMetadataSelected, setMetadataSelected] = useState<boolean>(false);
   const [isGisaidSelected, setGisaidSelected] = useState<boolean>(false);
   const [isGenbankSelected, setGenbankSelected] = useState<boolean>(false);
+  const state = store.getState();
+  const pathogen = selectCurrentPathogen(state);
```

Comment:
Since this is inside a react component, you should use `useSelector`. You can see an example of this in `src/views/Upload/components/Review/components/Table/index.tsx`